tools/Google.Cloud.Tools.VersionCompat.Tests/ClassTest.cs

```diff
@@ -0,0 +1,136 @@
+﻿// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace Class
+{
+    namespace NoChange.A { public class C { public C() { } } }
```

Comment:
Why all the parameterless constructors? Can't we just let the compiler generate them?

Reply:
Originally it was to make it very clear that there *is* a public ctor; but I agree it's mostly noise. Removed.